Sources/SpeziAccount/AccountSetup.swift

```diff
@@ -189,7 +180,7 @@ public struct AccountSetup<Header: View, Continue: View>: View {
     ///   - continue: A custom continue button you can place. This view will be rendered if the AccountSetup view is
     ///     displayed with an already associated account.
     public init(
-        setupComplete: @escaping (AccountDetails) -> Void = { _ in },
+        setupComplete: @MainActor @escaping (AccountDetails) async -> Void = { _ in try? await Task.sleep(for: .seconds(2)) },
```

Comment:
Is this intentional? I would assume this works fine, if I provide an empty setupComplete closure as well, right? other than maybe some very minor UI glitches

Reply:
You are right, that doesn't really do what I intended it to achieve.
I removed the default value and added a default 2s delay for the loading screen (as we had before). We use that to give the parent view the possibility to navigate away without showing the existing account view.